controller/search_blackbox_test.go

```diff
@@ -568,6 +569,18 @@ func (s *searchControllerTestSuite) TestSearchQueryScenarioDriven() {
 		assert.Len(t, result.Data, 3+5+1) // resolved items + items in sprint2
 	})
 
+	s.T().Run("space=spaceID AND title=special with $SUBSTR", func(t *testing.T) {
+		filter := fmt.Sprintf(`
+				{"$AND": [
+					{"space":"%s"},
+					{"title": {"$SUBSTR":"%s"}}
+				]}`,
+			spaceIDStr, "special")
+		_, result := test.ShowSearchOK(t, nil, nil, s.controller, &filter, nil, nil, nil, nil, &spaceIDStr)
+		require.NotEmpty(t, result.Data)
+		assert.Len(t, result.Data, 3)
```

Comment:
Please give all work items a title to make sure the `special` word doesn't appear in any WIs with idx > 2.

Reply:
Fixed. fd55350a2ddb68d7a25f931ca7e53758c2527c7b